crates/ef-testing-proc-macro/src/utils.rs

```diff
@@ -0,0 +1,37 @@
+use std::path::{Path, PathBuf};
+
+use crate::dir_reader::PathWrapper;
+
+/// Converts the path to a string and removes the "BlockchainTests" folder
+///
+/// # Example
+///
+/// Input: BlockchainTests/GeneralStateTests/stCallCreateCallCodeTest/Call1024PreCalls.json
+/// Output: GeneralStateTests/stCallCreateCallCodeTest/Call1024PreCalls.json
+pub(crate) fn blockchain_tests_to_general_state_tests_path(path: PathWrapper) -> PathWrapper {
+    Into::<PathBuf>::into(path)
+        .components()
+        .filter(|x| !x.as_os_str().eq_ignore_ascii_case("BlockchainTests"))
+        .collect::<PathBuf>()
+        .into()
+}
+
+/// Converts the path to a Vector of Strings
+pub(crate) fn path_to_vec_string(path: &Path) -> Result<Vec<String>, eyre::Error> {
+    path.iter()
+        .map(|os_str| {
```

Comment:
this is more like `relative_to`, for example in python https://docs.python.org/3/library/pathlib.html#pathlib.PurePath.relative_to